### DISCUSSION TITLE
Reproducible `targets` project

### DISCUSSION THREAD

**Discussion by shirdekel** [ropensci/targets#465] [Help] [answered]
I want to make sure that targets don't need to rerun after cloning the repository that house the project. You told me [here](https://github.com/ropensci/drake/issues/1353#issuecomment-759518321) that `targets` is meant to do this better than `drake`. Is it just the `_targets/objects/` directory that is needed to be kept for this? Or would I need to keep all of `_targets` under version control?

**Comment by wlandau** [comment, marked as answer]
`_targets/objects/` and `_targets/meta/meta`  are enough. There are other files in `_targets/meta/meta`, but those can be ignored.
